v2/withdraw_service.go

```diff
@@ -72,7 +72,7 @@ func (s *CreateWithdrawService) Name(v string) *CreateWithdrawService {
 func (s *CreateWithdrawService) Do(ctx context.Context) (*CreateWithdrawResponse, error) {
 	r := &request{
 		method:   "POST",
-		endpoint: "/wapi/v3/withdraw.html",
+		endpoint: "/sapi/v1/capital/withdraw/apply",
 		secType:  secTypeSigned,
 	}
 	r.setParam("asset", s.asset)
```

Comment:
https://binance-docs.github.io/apidocs/spot/en/#withdraw-sapi

Seems `asset` changed to `coin`